.kokoro/dependencies.sh

```diff
@@ -27,6 +27,16 @@ source ${scriptDir}/common.sh
 java -version
 echo $JOB_TYPE
 
+echo "****************** DEPENDENCY LIST COMPLETENESS CHECK *******************"
```

Comment:
This needs to run after the `mvn install` otherwise there won't be a flattened pom.xml

Reply:
ah that's correct.